docs/src/content/docs/da/environmental-impact.md

```diff
@@ -0,0 +1,128 @@
+
+```
+Cache-Control: public, max-age=604800, immutable
+```
+
+Hvordan man konfigurerer caching afhænger af din webhost. For eksempel anvender Vercel denne caching-strategi for dig uden behov for konfiguration, mens du kan sætte [tilpassede headers for Netlify][ntl-headers] ved at tilføje en `public/_headers`-fil til dit projekt:
+
+```
+/_astro/*
+  Cache-Control: public
+  Cache-Control: max-age=604800
+  Cache-Control: immutable
+```
+
+[cache]: https://csswizardry.com/2019/03/cache-control-for-civilians/
+[ntl-headers]: https://docs.netlify.com/routing/headers/
+
+## Strømforbrug
+
+Hvordan en webside er bygget, kan påvirke, hvor meget strøm det kræver at køre den på en brugers enhed. Ved at bruge minimalt JavaScript reducerer Starlight den mængde behandlingskraft, som en brugers telefon, tablet eller computer har brug for til at indlæse og gengive sider.
+
+Vær opmærksom på, når du tilføjer funktioner som analytik-tracking scripts eller JavaScript-tungt indhold som videoindlejringer, da disse kan øge sideens strømforbrug. Hvis du har brug for analytik, kan du overveje at vælge en letvægtsløsning som [Cabin][cabin], [Fathom][fathom] eller [Plausible][plausible]. Indlejringer som YouTube- og Vimeo-videoer kan forbedres ved at vente med at [indlæse videoen ved brugerinteraktion][lazy-video]. Pakker som [`astro-embed`][embed] kan hjælpe med almindelige tjenester.
+
+:::tip[Vidste du?]
+Parsing og kompilering af JavaScript er en af de mest ressourcekrævende opgaver, som browsere skal udføre. Sammenlignet med at gengive et JPEG-billede af samme størrelse kan [JavaScript tage mere end 30 gange længere tid at behandle][cost-of-js].
+:::
+
+[cabin]: https://withcabin.com/
+[fathom]: https://usefathom.com/
+[plausible]: https://plausible.io/
+[lazy-video]: https://web.dev/iframe-lazy-loading/
+[embed]: https://www.npmjs.com/package/astro-embed
+[cost-of-js]: https://medium.com/dev-channel/the-cost-of-javascript-84009f51e99e
+
+## Hosting
+
+Hvor en webside er hostet, kan have stor indflydelse på, hvor miljøvenlig din dokumentationsside er. Datacentre og serverfarme kan have en stor økologisk indflydelse, herunder højt elforbrug og intensiv brug af vand.
+
+At vælge en host, der bruger vedvarende energi, vil betyde lavere CO₂-udledninger for dit site. [Green Web Directory][gwb] er et værktøj, der kan hjælpe dig med at finde hostingfirmaer.
+
+[gwb]: https://www.thegreenwebfoundation.org/directory/
+
+## Sammenligninger
+
+Nysgerrig efter at vide, hvordan andre dokumentationsrammer klarer sig? Disse tests med [Website Carbon Calculator][wcc] sammenligner lignende sider bygget med forskellige værktøjer.
+
+| Framework                   | CO₂ per sidevisning | Rating |
```

Comment:
Looks like the table headings have been partially translated, e.g. rating? Is this expected in this case?